error[E0308]: arguments to this function are incorrect
  --> $DIR/tuple-arity-mismatch.rs:6:13
   |
LL |     let y = first ((1,2.0,3));
   |             ^^^^^^^---------^
   |                    |
   |                    expected `(isize, f64)`, found `(isize, f64, {integer})`
   |
note: function defined here
  --> $DIR/tuple-arity-mismatch.rs:3:4
   |
LL | fn first((value, _): (isize, f64)) -> isize { value }
   |    ^^^^^ ------------------------
help: provide an argument of the correct type
   |
LL |     let y = first({(isize, f64)});
   |             ^^^^^^^^^^^^^^^^^^^^^

error[E0308]: arguments to this function are incorrect
  --> $DIR/tuple-arity-mismatch.rs:12:13
   |
LL |     let y = first ((1,));
   |             ^^^^^^^----^
   |                    |
   |                    expected `(isize, f64)`, found `(isize,)`
   |
note: function defined here
  --> $DIR/tuple-arity-mismatch.rs:3:4
   |
LL | fn first((value, _): (isize, f64)) -> isize { value }
   |    ^^^^^ ------------------------
help: provide an argument of the correct type
   |
LL |     let y = first({(isize, f64)});
   |             ^^^^^^^^^^^^^^^^^^^^^

error: aborting due to 2 previous errors

For more information about this error, try `rustc --explain E0308`.
